docs/user_guide/README.md

```diff
@@ -51,17 +51,48 @@ the generated project and use the normal QGIS editing tools.
     * `Orphan metadata` displays tables not existing in the database itself, but the metadata exists in the
     `dataset` table.
     * `Orphan tables` is the reverse. It displays tables existing in the database but in the `dataset` table.
-1. On the `Dataset` table, open the attribute table, switch on **Edition** mode and add a new row.
+
+### Editing metadata
+
+Before creating metadata for our layer/table, you need to implement some table. To edit this table you need to 
+open the attribute table, switch on **Edition** mode and add a new row.
+
+Switching to edit mode:
+![Attribute table](../attribute_table_edit_mode.png)
+
+Adding a new row:
+![Attribute table](../attribute_table_new_row.png)
+
+1. On the `Theme` table you can add many theme corresponding to different theme of your layers.
+    You need to enter a **code** eg. "A01" and a **label** corresponding at the name eg. "Naturals area".
```

Comment:
Minor question, @mdouchin Shouldn't we show codes like `ENV` for environment, `CLI` for climatology?